Assets/Scripts/ManagerScripts/CountdownManager.cs

```diff
@@ -17,40 +17,52 @@ private void Awake()
         public TMP_Text countdownText;
         public GameObject countdownUI;
         public AudioSource countdownSound;
-        private bool _shouldCountTime;
+        public bool shouldCountTime;
         private bool _soundPlayed;
+        private bool _gameStart;
 
         private void Start()
         {
-            _shouldCountTime = SceneManager.GetActiveScene().name != "MainMenuScene";
+            shouldCountTime = true;
             countdown = 5f;
+            _gameStart = true;
         }
         
         private void Update()
         {
-            if ( _shouldCountTime && !_soundPlayed)
+            if (shouldCountTime)
             {
-                countingDown = true;
-                countdownSound.Play();
-                _soundPlayed = true;
-            }
-            
-            if (_shouldCountTime && countdown > 0)
-            {
-                countdown -= Time.deltaTime;
-                countdownText.text = (countdown).ToString("0");
-            }
-            else
-            {
-                countdownSound.Stop();
-                countdownUI.SetActive(false);
-                countingDown = false;
+                if (!_soundPlayed)
+                {
+                    countingDown = true;
+                    countdownSound.Play();
+                    _soundPlayed = true;
+                }
+
+                if (countdown > 0)
+                {
+                    SubtractTime(_gameStart);
+                    countdownText.text = (countdown).ToString("0");
+                }
+                else
+                {
+                    countdownSound.Stop();
+                    countdownUI.SetActive(false);
+                    _soundPlayed = false;
+                    _gameStart = false;
+                    countingDown = false;
```

Comment:
I think you should set shouldCountDown to false after line 53 (because we're done counting down and it shouldn't count down anymore)

Reply:
fixed